javascript/javascript1/week1/age.js

```diff
@@ -0,0 +1,69 @@
+let yearOfBirth = 1995;
+let yearFuture = 2027;
+let age = yearOfBirth - yearFuture;
+console.log("You will be " + age + "years old in " + yearFuture + ".");
+
+let dogYearOfBirth = 2010;
+let dogYearFuture = 2021;
+let dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears == true) {
+  console.log("Your dog will be " + dogYear + "years old in" + dogYearFuture);
+} else {
+  console.log("Your dog will be " + age + "years old in" + dogYearFuture);
+}
+
+let width;
+let depth;
+let height;
+let volumeInMeters = width * depth * height;
+let gardenSizeInM2;
+let fairHousePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+let actualHousePrice;
+
+if (actualHousePrice >= fairHousePrice) {
```

Comment:
`actualHousePrice `is not initialised (you never gave it a value), so the expression on line 25 is always false. You need to give your variables values or else you can't test if your code is working or not :)